### DISCUSSION TITLE
Is it possible to create multi-color label?

### DISCUSSION THREAD

**Discussion by Swami-Laxmikant** [patrykandpatrick/vico#950] [Questions] [answered]
### Question

<img width="531" alt="image" src="https://github.com/user-attachments/assets/244a185b-e87d-4d3e-9ee2-7838ccf5e0b8">

I’m interested in creating a graph similar to this one. I can manage most aspects, but I’d like to know if it’s possible to customize the color of each label of the left axis item individually.

### Vico version(s)

any

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello, @Swami-Laxmikant. Yes, this is possible. You can do it via `CartesianValueFormatter`. It returns a `CharSequence`, which means you can use `SpannableString` to color labels individually.
